templates/alignment_summary.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
why are these files in in template and not in bin? Also, I think there is enough overlap to combine them to one, maybe?

Reply:
I put them in a template to be able to use nextflow variables. I would prefer to have them separate as they output specific text depending on the case.